### DISCUSSION TITLE
Uba error in termux

### DISCUSSION THREAD

**Discussion by frank4532** [eliranwong/UniqueBible#995] [Q&A]
I tried to install Uba in termux. Termux now install python 3.10 by default. I add a screenshot.
Is there a way to install webtop Uba in android?
![Screenshot_2022-04-21_134240](https://user-images.githubusercontent.com/85790869/164453185-b789af15-8e1d-4661-85ec-10c6872760c7.jpg)

**Comment by eliranwong** [comment]
Hi, I use Termux to run UBA too.  I managed to run UBA http-server on my Android devices.  You may refer to the following wiki pages about it:

Http-server / web version: https://github.com/eliranwong/UniqueBible/wiki/Web-Version-%5Bavailable-OFFLINE%5D

Setup audio bible: https://github.com/eliranwong/UniqueBible/wiki/Set-up-bible-audio-on-Http-server

Please note that Termux does not support docker.  Therefore, webtop version is not supported on Termux.  You may check our recommendations about UBA setup on different platforms:

https://github.com/eliranwong/UniqueBible/wiki/Recommendations-for-Different-Platforms

**Reply by frank4532** [reply]
Hi,
Thanks for the answer, but Uba is not working on my Android device. I tried to reinstall but I have the same message you can see on the screenshot. When I open http://localhost:8080/ I have the error message the site can't be reached. For the installation I followed your instructions on wiki pages.
